r_basics_introduction/r_basics_introduction.md

```diff
@@ -253,7 +254,24 @@ style = "border: 1px solid rgb(var(--color-highlight));"-->
 ![RStudio as shown in the cloud platform Binder. The left side of the screen displays the "Console" tab. On the right there is a split between an "upper" and "lower" section. The upper section is the "Environment", which is empty. The lower section, "files", shows one file (environment.yml) and three folders (r_basics_introduction, r_basics_transform_data, r_basics_visualize_data)](media/binder_rstudio.png)<!--
 style = "border: 1px solid rgb(var(--color-highlight)); max-width: 800px;"-->
 
-<h3>Option 2: Work on Your Computer</h3>
+<h3>Option 2: Use Posit Cloud</h3>
+
+Posit (the company formerly known as RStudio) provides a multi-tiered cloud environment for using RStudio.  Unlike option 1 above, this option does require you to have an account with Posit Cloud, their online RStudio server.  The good news is that the base level of Posit Cloud is free!
+
+First, you'll need to [create a (free!) Posit cloud account](https://posit.cloud/plans).  
+
+Then, once you're logged in at https://posit.cloud, open the "education\_r\_environment" project at https://posit.cloud/content/5273350.  That will give you a temporary copy so you can run our code, but not make any changes to it.
```

Comment:
The link to posit.cloud doesn't work for some reason. This might help? Maybe? 
```suggestion
Then, once you're logged in at https://posit.cloud/, open the "education\_r\_environment" project at https://posit.cloud/content/5273350.  That will give you a temporary copy so you can run our code, but not make any changes to it.
```